### ISSUE TITLE
Remove/destroy resources

### ISSUE THREAD

**Issue by geovanisouza92** [flynn/flynn#1413]
We need an API endpoint for remove and destroy resources (e. g. PostgreSQL db), something like

```sh
$ flynn resource
ID                                Provider ID                       Provider Name
3eb13704c1e74697a5fb542bdee189e6  91718ec7103f4525bb77ffabf15e355b  postgres

$ flynn resource remove 3eb13704c1e74697a5fb542bdee189e6
Resource 3eb13704c1e74697a5fb542bdee189e6 (postgres provider) removed from app.
```

**Comment by shideneyu** [comment]
How can we remove a resource for the time being, without that feature ?

**Comment by geovanisouza92** [comment]
We can't.

**Comment by shideneyu** [comment]
I added two times my DB by mistake.
I'm recreating my entire cluster for this mistake.

**Comment by lmars** [comment]
Please note this is being implemented in #1533 which should hopefully land soon.
